### ISSUE TITLE
HtCookies

### ISSUE THREAD

**Issue by yegor256** [yegor256/cactoos-http#8]
Let's create class `HtCookies` (implements `Map<String,String>`) to read cookies from HTTP response:

```java
String cookie = new HtCookies(
  new HtHead(new HtResponse("http://www.google.com"))
).get("user");
```

**Comment by 0crat** [comment]
Job [#8](https://github.com/yegor256/cactoos-http/issues/8) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#8](https://github.com/yegor256/cactoos-http/issues/8) assigned to @vssekorin[/z](https://www.0crat.com/u/vssekorin), here is [why](https://www.0crat.com/footprint/C63314D6Z/941335176); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos-http#8` already assigned to @vssekorin, can't assign to @vssekorin

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#45](https://github.com/yegor256/cactoos-http/issues/45), [#46](https://github.com/yegor256/cactoos-http/issues/46) are still not solved.

**Comment by VsSekorin** [comment]
@yegor256 please close issue. PR #17 was merged

**Comment by yegor256** [comment]
@VsSekorin thanks!

**Comment by elenavolokhova** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @vssekorin[/z](https://www.0crat.com/u/vssekorin)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#46](https://github.com/yegor256/cactoos-http/issues/46) is still not solved; solved: [#45](https://github.com/yegor256/cactoos-http/issues/45).

**Comment by 0pdd** [comment]
@yegor256 all 2 puzzles are solved here: [#45](https://github.com/yegor256/cactoos-http/issues/45), [#46](https://github.com/yegor256/cactoos-http/issues/46).
